warehouse/templates/includes/accounts/edit-profile-button.html

```diff
@@ -14,4 +14,16 @@
 
 {% if request.user == user %}
   <a href="{{ request.route_path('manage.account') }}" class="button button--primary author-profile__edit-button">Edit profile</a>
+
+  <div class="sidebar-section">
+    <div style="height:20px"></div>
+    <h3 class="sidebar-section__title">Statistics</h3>
+    <p>View statistics for your projects via <a href="https://libraries.io/">Libraries.io</a>, or by using <a href="https://packaging.python.org/guides/analyzing-pypi-package-downloads/">Google BigQuery</a></p>
+  </div>
+{% else %}
+<div class="sidebar-section">
+  <div style="height:20px"></div>
```

Comment:
The lint violation is due to the fact that our Content Security Policies do not allow inline style. You'll need to update the SCSS in https://github.com/pypa/warehouse/blob/053c890c8b305cb5ca03de718866ba600915ab40/warehouse/static/sass/blocks/_sidebar-section.scss#L27 or find an appropriate class to apply.

Reply:
Also note that the linter can be run locally with `make lint`